Lombiq.OrchardCoreApiClient/ApiClient.cs

```diff
@@ -74,6 +72,11 @@ public void Dispose()
 
     protected virtual void Dispose(bool disposing)
     {
+        if (!_lazyOrchardCoreApi.IsValueCreated) return;
+
+        object api = _lazyOrchardCoreApi.Value;
+        (api as IDisposable)?.Dispose();
```

Comment:
When can it be disposable?

Reply:
My bad, I was just testing if it's needed and forgot to remove it afterwards.